tools/install/hcledit.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eo pipefail
+# shellcheck disable=SC1091 # Created by Dockerfile above script call
+source /.env
+
+# hcledit
+if [[ $HCLEDIT_VERSION != false ]]; then
+  HCLEDIT_RELEASES="https://api.github.com/repos/minamijoyo/hcledit/releases"
+
+  if [[ $HCLEDIT_VERSION == latest ]]; then
+    curl -L "$(curl -s ${HCLEDIT_RELEASES}/latest | grep -o -E -m 1 "https://.+?_${TARGETOS}_${TARGETARCH}.tar.gz")" > hcledit.tgz
```

Comment:
1. Do we indeed want `curl` to produce output except errors? If not, then:
```suggestion
    curl -sSL "$(curl -s ${HCLEDIT_RELEASES}/latest | grep -o -E -m 1 "https://.+?_${TARGETOS}_${TARGETARCH}.tar.gz")" > hcledit.tgz
```
2. It makes no sense to use tool-specific names:
```suggestion
    curl -L "$(curl -sSL ${RELEASES}/latest | grep -o -E -m 1 "https://.+?_${TARGETOS}_${TARGETARCH}.tar.gz")" > release.tgz
```

ps: curl's `-SsL` and grep's `-o -E -m 1` might probably be converted into vars.

Reply:
>Do we indeed want curl to produce output except errors? If not, then:

Yeah, why not? Let users know of download status, as it's time-consuming operation

Reply:
Ah, okay. Not sure whether curl's progress bar is shown correctly within Docker's build output though.

Reply:
Btw, I double check, that's different case
```
curl -sSL ${RELEASES}/latest | grep -o -E -m 1 "https://.+?_${TARGETOS}_${TARGETARCH}.tar.gz"
```

must return string of artifact path
 and then `curl -L` will download it. 
 
 So yes, there should be output

Reply:
> So yes, there should be output

The `-s` and `-S` is not about suppressing regular output (http body), but about suppressing curl's output (`-s` to make it silent and `-S` to still show errors).
Also the `-L` is good to be added to curl's opts as it ensures http redirects are followed.